src/Test/Perf/helloworld/HelloWorld.csx

```diff
@@ -0,0 +1,20 @@
+#load "../util.csx"
+using System.IO;
+
+var pathToCsc = Path.Combine(BinDebugDirectory(), "csc.exe");
+var pathToHelloWorld = Path.Combine(MyWorkingDirectory(), "HelloWorld.cs");
+var pathToOutput = Path.Combine(MyArtifactsDirectory(), "HelloWorld.exe");
+
+ProcessResult result;
+
+var msToCompile = WalltimeMs(out result,
```

Comment:
In my mental model, the runner.csx would be the thing iterating, but sticking with the idiom here, rather than ```WalltimeMs```, I can imagine calling ```Iterate(() => ShellOut(...```  Iterate could be starting/stopping profiling for each iteration.  Assuming it isn't too slow to collect/parse the traces, then this would naturally give us the granular results you want (total process time, thread time, cpu time, etc).  We could have a "-noprofile" switch if you just want to run the test to validate functionality, but I really don't want to be in the state we are today where we have these numbers with no profile data to back them up (so you have to try and get a live repro before you can investigate).